### PR TITLE
Modify readme to link to updated documentation endpoint

### PR DESCRIPTION
**Pull request by sblaurock**
Also added new badge to indicate minzipped filesize.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,292 +31,4 @@ var constructorio = new ConstructorIOClient({
...
-| `variation_id` | string | |
-| `section` | string | |
-| `result_id` | string | |
-| `result_count` | integer | |
-| `result_page` | integer | |
-| `result_position_on_page` | integer | |
-| `num_results_per_page` | integer | |
-| `selected_filters` | string or array | |
-
-#### Receive status of tracking requests
```
Comment (jjl014) [inline README.md:299]: Should we leave this in or put it somewhere in the JSDocs?

File: README.md [hunk 2]
```diff
@@ -30,292 +31,4 @@ var constructorio = new ConstructorIOClient({
...
-The status of tracking requests can be observed through emitted events. `messageType` must be either "success" or "error", and the `callback` parameter must be a function that will be passed the message data as the first and only argument.
-```javascript
-constructorio.tracker.on(messageType, callback);
-```
-
-#### Dispatched events
```
Comment (jjl014) [inline README.md:305]: Same with this

**Review by jjl014 — COMMENTED** [review]
Lgtm, left some comments.

File: README.md (hunk 1 above)
Comment (sblaurock) [reply README.md:299]: It's there! https://constructor-io.github.io/constructorio-client-javascript/module-Tracker.html#~on

File: README.md (hunk 2 above)
Comment (sblaurock) [reply README.md:305]: This one we might want to leave in the readme - not sure how we'd include this concept in the JSDocs as it's a bit outside of core code.

File: README.md [hunk 3]
```diff
@@ -30,292 +31,4 @@ var constructorio = new ConstructorIOClient({
...
-}, false);
-```
-
-## Development / npm commands
-
-```bash
-npm run lint          # run lint on source code and tests
-npm run test          # run tests
-npm run coverage      # run tests and serves coverage reports from localhost:8081
-npm run docs          # build and serve documentation from localhost:8082
```
Comment (sblaurock) [inline README.md:320]: We should leave these in the readme as well

File: README.md (hunk 1 above)
Comment (jjl014) [reply README.md:299]: Nice!

File: README.md (hunk 2 above)
Comment (jjl014) [reply README.md:305]: Cool, yea. Sounds good to me.

File: README.md (hunk 3 above)
Comment (jjl014) [reply README.md:320]: Sounds good 👍

**Review by jjl014 — APPROVED** [review]
🚢 🚀

**Pull request merged**